gtsfm/scene_optimizer.py

```diff
@@ -56,8 +55,21 @@
 from gtsfm.frontend.verifier.verifier_base import VerifierBase
 from gtsfm.loader.folder_loader import FolderLoader
 
-# configure loggers to avoid DEBUG level stdout messages
-logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+def get_logger():
```

Comment:
Thanks for the PR, Ayush. I think we define this `get_logger()` function in multiple places, so it could probably go in `gtsfm/utils/logger_utils.py` or something